UnityProjects/MRTKDevTemplate/Assets/Scenes/NearMenuExamples.unity

```diff
@@ -2641,7 +2645,7 @@ PrefabInstance:
       objectReference: {fileID: 0}
     - target: {fileID: 8110664055982005406, guid: f443144a6d408c34bb8a7d70a73644c4, type: 3}
       propertyPath: m_AnchoredPosition.x
-      value: 0
+      value: 84
       objectReference: {fileID: 0}
```

Comment:
@ms-RistoRK  please double check these changes.   Theis scene is changing more than just enabling the speech interactor.

Reply:
Double checked the scene, those extra changes in file were introduced by Unity Editor.  Just in case I repeated the process of copying .unity file from feature/XRI3 branch latest commit -> double-checked the new controllerless rig is used -> enabled MRTKSpeech rig component.
Resolving.